.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -35,20 +28,24 @@ body:
       render: python
     validations:
       required: true
-  - type: textarea
-    id: problem
-    attributes:
-      label: Issue Description
-      description: Please provide a description of the issue shown in the reproducible example.
-    validations:
-      required: true
   - type: textarea
     id: expected-behavior
     attributes:
       label: Expected Behavior
       description: Please describe or show a code example of the expected behavior.
     validations:
       required: true
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Bug report checklist
+      options:
+        - label: I have checked that this issue has not already been reported.
+          required: true
+        - label: I have confirmed this bug exists on the [latest release](https://github.com/slundberg/shap/releases) of shap.
+          required: true
+        - label: I have confirmed this bug exists on the master branch of shap.
```

Comment:
Could we include `pip install git+...` instructions for installing from the master branch? Some users may not know how to install from master.

Reply:
I added a section to the contributing guidelines. Perhaps in future we could migrate this information to readthedocs; but I think this is simpler for a first-pass!